armi/reactor/blueprints/blockBlueprint.py

```diff
@@ -170,6 +179,46 @@ def construct(
                 runLog.warning(str(e), single=True)
         return b
 
+    def _checkByComponentMaterialInput(self, materialInput):
```

Comment:
Can we have a unit test for this method?

Reply:
Added.